cmd/sgx-device-plugin/main.go

```diff
@@ -12,7 +15,24 @@ import (
 	"github.com/AliyunContainerService/sgx-device-plugin/pkg/utils"
 )
 
+func usage() {
+	fmt.Fprintf(os.Stdout, `sgx-device-plugin
+Usage: sgx-device-plugin
+
+Options:
+`)
+	flag.PrintDefaults()
+}
+
+func init() {
+	flag.BoolVar(&sgx.EnableAESMSocketAttach, "enable-aesm-socket-attach", false, "Enables attachment of AESM service socket")
+	flag.StringVar(&sgx.AESMSocket, "aesm-socket", "/var/run/aesmd", "Path of AESM socket")
```

Comment:
"/var/run/aesmd" => "/var/run/aesmd/aesm.socket"
Mount directory of this socket(path.Dir(sgx.AESMSocket)).

Reply:
As discussed, we eventually decided to remove this option since aesm-socket path cannot be specified by users for now.